docs/reference/garbage-collection.md

```diff
@@ -191,20 +191,40 @@ spark-submit --class io.treeverse.clients.GarbageCollector \
 
 The list of expired objects is written in Parquet format in the storage
 namespace of the bucket under
-`_lakefs/retention/gc/addresses/run_id=RUN_ID`, where RUN_ID identifies the
+`_lakefs/retention/gc/addresses/mark_id=MARK_ID`, where MARK_ID identifies the
```

Comment:
Do we need a bc note for runs on an older version?

Reply:
yes, thanks - added